spec/companies_house/client_spec.rb

```diff
@@ -160,36 +174,39 @@
         expect(response).to eq(%w[item1 item2])
       end
 
+      # rubocop:disable RSpec/MultipleExpectations
```

Comment:
This is fine - splitting this into separate tests would be rubbish, I don't think we should mind disabling the cop here.